app/models/person.rb

```diff
@@ -1,6 +1,21 @@
 # frozen_string_literal: true
-
 class Person < VersionedModel
+  FEED_ATTRIBUTES = %w[
+    id
```

Comment:
I'm sure they'll come back to us with something about anonymising these but that can be a separate PR :).